### PR TITLE
Fix some redirects

### PR DESCRIPTION
**Pull request by gforcada**

### CODE REVIEW

File: deployment/ansible.rst [hunk 1]
```diff
@@ -3,7 +3,7 @@ Intro To Ansible
 ================
 
 `Ansible <https://www.ansible.com/>`_ is an open-source configuration management, provisioning and application deployment platform
-written in Python and using `YAML <http://yaml.org/start.html>`_ (YAML Ain't Markup Language) as a configuration language.
+written in Python and using `YAML <https://yaml.org/start.html>`_ (YAML Ain't Markup Language) as a configuration language.
```
Comment (stevepiercy) [inline deployment/ansible.rst]: Suggest https://yaml.org/about.html instead.

File: mastering-plone/ide.rst [hunk 2]
```diff
@@ -21,7 +21,7 @@ People pick editors themselves. Use whatever you are comfortable and productive
 * `Wing IDE <http://wingide.com/>`_
 * vim
 * emacs
-* `PyDev <http://www.pydev.org/>`_ for `Eclipse <http://www.eclipse.org/>`_
+* `PyDev <http://www.pydev.org/>`_ for `Eclipse <https://www.eclipse.org/>`_
```
Comment (stevepiercy) [inline mastering-plone/ide.rst]: Please use https://www.pydev.org/

File: mastering-plone/plone5.rst [hunk 3]
```diff
@@ -21,7 +21,7 @@ It is a Diazo theme, meaning it uses :py:mod:`plone.app.theming` to insert the o
 
 It uses html5, so it uses ``<header>``, ``<nav>``, ``<aside>``, ``<section>``, ``<article>`` and ``<footer>`` for semantic html.
 
-The theme is mostly built with `LESS <http://lesscss.org/>`_ (lots of it!) and uses the same grid system as `bootstrap <http://getbootstrap.com/css/#grid>`_. This means you can use css classes like ``col-xs-12 col-sm-9`` to control the width of elements for different screen-sizes. If you prefer a different grid-system (like `foundation <https://foundation.zurb.com/sites/docs/grid.html>`_) over bootstrap you can adapt the theme to use that.
+The theme is mostly built with `LESS <http://lesscss.org/>`_ (lots of it!) and uses the same grid system as `bootstrap <https://getbootstrap.com/css/#grid>`_. This means you can use css classes like ``col-xs-12 col-sm-9`` to control the width of elements for different screen-sizes. If you prefer a different grid-system (like `foundation <https://foundation.zurb.com/sites/docs/grid.html>`_) over bootstrap you can adapt the theme to use that.
```
Comment (stevepiercy) [inline mastering-plone/plone5.rst]: - One sentence per line.
- s/bootstrap/Bootstrap (2 times)
- s/css/CSS
- s/screen-sizes/screen sizes
- s/grid-system/grid system
- s/foundation/Foundation

**Comment by gforcada** [comment]
@stevepiercy better late than never :+1: fixed all the comments you made on the changes. It should be ready to review and merge :smile:

**Comment by stevepiercy** [comment]
Thank you!

**Pull request merged**